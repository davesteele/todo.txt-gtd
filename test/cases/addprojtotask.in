# AProject
An AProject task @frobnitz

# _None
