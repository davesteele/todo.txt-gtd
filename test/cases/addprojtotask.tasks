
@frobnitz
An AProject task @frobnitz
