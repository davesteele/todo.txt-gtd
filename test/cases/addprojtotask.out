# AProject
An AProject task @frobnitz +AProject

# _None
